contributing_to_docs/tools_and_setup.adoc

```diff
@@ -82,6 +82,12 @@ The following instructions describe how to install all the required tools to do
 live content editing on a Fedora Linux system.
 
 1. Install the _RubyGems_ package with `yum install rubygems`
+
+NOTE: An alternative to installing rubygems using yum is to use the rvm method. On certain systems, the yum method installs an older version which can cause issues. If you are having issues, try the following, which has been taken from the link:https://rvm.io/rvm/install[RVM site].
```

Comment:
You could reverse the order of this paragraph to state the issue first and then the workaround. For example:

"NOTE: On certain systems, YUM installs an older version of RubyGems that can cause issues. As an alternative, you can install RubyGems by using RVM. The following example is referenced from the link:https://rvm.io/rvm/install[RVM site]:"

I have also introduced some capitalisation into that example. I am not entirely sure whether YUM should be fully capitalised, since examples vary across resources online. An alternative could be to place `yum` and `rvm` in back ticks using the lowercase styling, to refer to the commands directly.